library/Director/Integration/Icingadb/IcingadbBackend.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Icinga\Module\Director\Integration\Icingadb;
+
+use Icinga\Application\Modules\Module;
+use Icinga\Module\Director\Auth\Permission;
+use Icinga\Module\Director\Auth\Restriction;
+use Icinga\Module\Director\Integration\BackendInterface;
+use Icinga\Module\Icingadb\Common\Auth;
+use Icinga\Module\Icingadb\Common\Database;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Icingadb\Model\Service;
+use Icinga\Web\Url;
+use ipl\Orm\Query;
+use ipl\Stdlib\Filter;
+
+class IcingadbBackend implements BackendInterface
+{
+    use Database;
+    use Auth;
+
+    /** @var bool */
+    protected $isAvailable;
```

Comment:
That's only used internally and without Icinga DB Web being enabled, this class isn't instantiated to begin with. So please remove this property.